aws/cloudformation/standalone/marketing/deployment/marketing.yml.erb

```diff
@@ -511,102 +540,37 @@ Resources:
           GeoRestriction:
             RestrictionType: none
 
+        CacheBehaviors:
+          # Path: /api/*
+          # Description: Marketing application API routes
+          - AllowedMethods:
+              - GET
+              - HEAD
+              - OPTIONS
+              - PUT
+              - POST
+              - PATCH
+              - DELETE
+            CachePolicyId: 4135ea2d-6df8-44a3-9df3-4b5a84be39ad # Caching Disabled (APIs are not cached)
+            OriginRequestPolicyId: 216adef6-5c7f-47e4-b989-5492eafa07d3 # AllViewer
+            PathPattern: /api/*
+            TargetOriginId: marketing-sites-application-origin
+            ViewerProtocolPolicy: redirect-to-https
+          # Path: /_next/*
+          # Description: Marketing _application_ static assets
+          - AllowedMethods:
+              - GET
+              - HEAD
+              - OPTIONS
+            CachePolicyId: 4135ea2d-6df8-44a3-9df3-4b5a84be39ad # Caching Disabled (caching is via the frontdoor cloudfront)
```

Comment:
I'm considering changing the cache policy in the Slack thread here instead of disabling it. This way the frontdoor utilzes "caching disabled" and caching behavior is defined in this cloudfront from the start. Let me know what you think. Same applies to the above behavior.